### PR TITLE
feat: :sparkles: add builder page for _indralukmana_

### PR DESCRIPTION
**Pull request by indralukmana**
## Description

Add personal builder page for _indralukmana_ with following points of considerations:
- as much as possible make the page server rendered / statically generated
- use existing library based on the `package.json` (mostly leveraging the existing React Icons lib)
- use the existing `Address` component from ScaffoldETH for ETH address related information
- lightweight client-side effects
- implemented some unique effects that I'm experimenting on my projects

The followings are sample screenshots of the page in mobile and desktop views, displayed in dark and light mode:

<details>
<summary>💻 Desktop Views</summary>

buidlguidl_batch8_indra-desktop-dark

buidlguidl_batch8_indra-desktop-light

</details>

<details>
<summary>📱 Mobile Views</summary>

buidlguidl_batch8_indra-mobile-dark

buidlguidl_batch8_indra-mobile-light

</details>

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

- https://github.com/BuidlGuidl/batch8.buidlguidl.com/issues/9

Your ENS/address: 0x80Ad2861Ab5D4EeA61330A4bd7d6969357C463C3

### CODE REVIEW

File: packages/nextjs/app/builders/0x80Ad2861Ab5D4EeA61330A4bd7d6969357C463C3/_components/IndraAboutMeSection.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const IndraAboutMeSection = () => {
+  return (
+    <section className="container mx-auto space-y-2">
```
Comment (phipsae) [inline packages/nextjs/app/builders/0x80Ad2861Ab5D4EeA61330A4bd7d6969357C463C3/_components/IndraAboutMeSection.tsx:3]: You could include a picture of yourself or an avatar to add a more personal and friendly touch.

**Review by phipsae — CHANGES_REQUESTED** [review]
Very nice written code and personal page @indralukmana! 
Really appreciate the effort.

I don't have much to add, just a few small suggestions below.

Thanks for the excellent PR!

File: packages/nextjs/app/builders/0x80Ad2861Ab5D4EeA61330A4bd7d6969357C463C3/page.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import IndraAboutMeSection from "~~/app/builders/0x80Ad2861Ab5D4EeA61330A4bd7d6969357C463C3/_components/IndraAboutMeSection";
+import IndraSkillsSection from "~~/app/builders/0x80Ad2861Ab5D4EeA61330A4bd7d6969357C463C3/_components/IndraSkillsSection";
+import IndraSocialsSection from "~~/app/builders/0x80Ad2861Ab5D4EeA61330A4bd7d6969357C463C3/_components/IndraSocialsSection";
+import IndraSparkleName from "~~/app/builders/0x80Ad2861Ab5D4EeA61330A4bd7d6969357C463C3/_components/IndraSparkleName";
+import { Address } from "~~/components/scaffold-eth";
+
+const BuilderPage: React.FC = () => {
```
[the remark this answers is not in the document]
Comment (indralukmana) [reply packages/nextjs/app/builders/0x80Ad2861Ab5D4EeA61330A4bd7d6969357C463C3/page.tsx]: Thanks! missed that type, NextPage is the proper type here. Also remind me of on my Todo for researching linter rules related to Next.js usages.

fixed at [`0acaedc` (#16)](https://github.com/BuidlGuidl/batch8.buidlguidl.com/pull/16/commits/0acaedc8890072002267c6d05b895a64908bdb0b)

File: packages/nextjs/app/builders/0x80Ad2861Ab5D4EeA61330A4bd7d6969357C463C3/_components/IndraAboutMeSection.tsx (hunk 1 above)
Comment (indralukmana) [reply packages/nextjs/app/builders/0x80Ad2861Ab5D4EeA61330A4bd7d6969357C463C3/_components/IndraAboutMeSection.tsx]: I actually don't have a lot of photo 😆 grabbing some recent one here. Was planning to get some user/profile NFT but haven't got a chance to look for something I'd like.

fixed at [`95b455c` (#16)](https://github.com/BuidlGuidl/batch8.buidlguidl.com/pull/16/commits/95b455c9eb185d7add941ff67d7bd8e182da0cb9)

**Comment by indralukmana** [comment]
Thank you for the reviews @phipsae! sent my fixes 🎉

**Comment by phipsae** [comment]
Very nice job! Appreciated, and merged :)

**Pull request merged**